js/tempConvert.js

```diff
@@ -0,0 +1,7 @@
+let fahrenheit = 98
+const celsius = fahrenheit * (5/9) - 32
+const kelvin = celsius + 273
+
```

Comment:
Math looks good but we need some conditionals dependent on the _type_ of temperature and what we want to convert it to. so set two variables. One for farenheit like you have and one for a letter that represents those math equations to use in the conditionals.